manifests/cfcr.yml

```diff
@@ -510,7 +538,14 @@ variables:
 - name: tls-etcdctl-v0-29-0
   options:
     ca: etcd_ca
-    common_name: etcdClient
+    common_name: root
+    extended_key_usage:
+    - client_auth
+  type: certificate
+- name: tls-etcdctl-flanneld-v0-29-0
```

Comment:
Cert shouldn't have `v0-29-0` on the end. That was only necessary because of cert rotation complications.

Reply:
Ok thanks. Removed